aspnetcore/security/authentication/facebook-logins.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Facebook external login setup
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/1/2016
+ms.topic: article
+ms.assetid: 8c65179b-688c-4af1-8f5e-1862920cda95
+ms.prod: aspnet-core
+﻿uid: security/authentication/facebook-logins
+---
+# Configuring Facebook authentication
+
+<a name=security-authentication-facebook-logins></a>
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Pranav Rastogi](https://github.com/rustd)
+
+This tutorial shows you how to enable your users to log in with their Facebook account using a sample ASP.NET Core project created in the [previous section](sociallogins.md). We start by creating a Facebook AppId by following the [official steps](https://developers.facebook.com/docs/apps/register).
+
+## Creating the app in Facebook
+
+*  Navigate to [https://developers.facebook.com/apps](https://developers.facebook.com/apps) and log in. If you don't already have a Facebook account, use the *sign up* link on the login page to create one.
+
+* Tap **My Apps** in the upper right corner:
+
+![image](sociallogins/_static/FBMyApps.png)
+
+* Tap **+ Add a New App** and fill out the form to create a new app ID:
+
+![image](sociallogins/_static/FBNewAppID.png)
```

Comment:
The filename should (maybe) be `FBNewAppId.png`. Chrome fails to load the image when I'm looking at the change.